src/router.js

```diff
@@ -11,5 +12,10 @@ export default new Router({
         path: '/',
         name: 'developers-list',
         component: DevelopersList
+    }, 
+    {
+        path: '/nominate',
+        name: 'nominate',
+        component: Nominate
```

Comment:
Use lazy loading for this route: [lazy-loading-routes](https://router.vuejs.org/guide/advanced/lazy-loading.html#lazy-loading-routes)

Reply:
Done .. .